### DISCUSSION TITLE
¿Qué es mejor? ¿Smartphones o computadoras?

### DISCUSSION THREAD

**Discussion by xpistaxo05** [hackatomik/SISCOx256#30] [Chill | Carrera]
Esta es una pregunta que tengo desde hace rato y la verdad no me decido la respuesta

**Comment by xpistaxo05** [comment]
en lo personal siento que los teléfonos son mejores porque pueden hacer basicamente lo mismo que las computadoras e incluso te caben en el bolsillo 🔥

**Comment by finalMadFox6325** [comment]
Pienso que las computadoras ofrecen mejores ventajas a la hora de hacer trabajos en el hogar y también el hacer tareas tanto escolares como Universitarias

**Reply by xpistaxo05** [reply]
como seria eso? desde mi punto de vista es lo mismo pq ambos dispositivos tienen acceso a un navegador y ambos tambien corren el word, powerpoint, y mas aplicaciones asi 😂

**Reply by luis0618** [reply]
Pienso que cada uno es diferente ya que cualquiera de los dos nos ayuda en alguna tarea o proyecto también depende de que actividad se vaya a realizar ya eso dependería de con cuál te sientas más cómodo al momento de trabajar

**Reply by xpistaxo05** [reply]
bueno cierto, no puedes usar un telefono para jugar elden ring 😂
pero ey, de q te hace las tareas te las hace sin problemas como lo haria una computadora

**Comment by FsZ0** [comment]
Siendo objetivos, el punto fuerte del celular es su portabilidad, realmente es dificil compararlos ya que sus funciones son parecidas pero a la vez distintas, sin contar que programas de alto peso como autocad usado por ingenieros, blender usado por artistas 3d entre otros no pueden ser montados y utilizados de manera correcta en un smartphone, sin embargo, tener una camara, calculadora, buscador web (entre otras muchas funciones) en el bolsillo todo el tiempo es algo que no se podia imaginar hace unos años, siento que el debate deberia estar mas dirigido en las funciones indviduales de cada uno mas que cual es mejor



la computadora es mejor igual 🗣️🗣️🗣️🗣️🗣️🗣️🗣️🗣️🗣️🗣️🗣️🗣️

**Comment by quantumquirkxyz** [comment]
Hola, bueno desde mi perspectiva, eso va acorde al usuario y sus requerimientos, ya que muchos prefieren sus Laptops y otros sus celulares, dependiendo de lo que hagan en su día a día, así que por lo menos para mi no hay un "mejor que". Yo soy un usuario de ambos dispositivos, así que no tengo preferencia, ya que los uso acorde a mis necesidades en el momento.

**Comment by AlejandraMichellx** [comment]
Depende de que ámbito hablemos, para cosas cotidianas como chatear, llamar o usar las redes lo mejor seria un celular ya que es fácil de llevar y practico, en cambio un celular se vuelve mas complicado usarlo a la hora de hacer trabajos académicos o profesionales como tablas de excel, informes, programar, etc. Donde es mejor en ese aspecto las computadoras.

**Reply by quantumquirkxyz** [reply]
Claro que si, por eso, el uso de smartphone o PC, varía dependiendo de tus necesidades como profesional
